ui/apps/moderation/src/components/dashboard/assign-admin.tsx

```diff
@@ -8,14 +8,14 @@ import Text from '@akashaorg/design-system-core/lib/components/Text';
 import ModeratorListItem from './moderator-list-item';
 import { IPageHeaderProps, PageHeader } from '../common';
 
-export interface IAssignAdminProps extends IPageHeaderProps {
+export type IAssignAdminProps = IPageHeaderProps & {
```

Comment:
The initial `I` wouldn't make sense when used with `type`

Reply:
gotcha. I will check all those again